ROMFS/px4fmu_common/init.d-posix/airframes/1036_malolo

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+#
+# @name Plane SITL
+#
+
+sh /etc/init.d/rc.fw_defaults
+
+if [ $AUTOCNF = yes ]
+then
+	param set EKF2_ARSP_THR 8
+	param set EKF2_FUSE_BETA 1
+	#param set EKF2_MAG_ACCLIM 0
+	#param set EKF2_MAG_YAWLIM 0
+
+	param set FW_LND_AIRSPD_SC 1.1
+	param set FW_LND_ANG 5
+	param set FW_THR_LND_MAX 0
+	param set FW_LND_HHDIST 30
+	param set FW_LND_FL_PMIN 9.5
+	param set FW_LND_FL_PMAX 20
+	param set FW_LND_FLALT 5
+	param set FW_LND_TLALT 15
+
+	param set FW_L1_PERIOD 25
+
+	param set FW_P_TC 0.4
+	param set FW_PR_FF 0.40
+	param set FW_PR_I 0.05
+	param set FW_PR_P 0.05
+
+	param set FW_R_TC 0.45
+	param set FW_RR_FF 0.40
+	param set FW_RR_I 0.132
+	param set FW_RR_P 0.085
+
+	param set FW_W_EN 1
+
+	param set MIS_LTRMIN_ALT 30
+	param set MIS_TAKEOFF_ALT 20
+	param set MIS_DIST_1WP 2500
+	param set MIS_DIST_WPS 10000
+
+	param set NAV_ACC_RAD 15
+	param set NAV_DLL_ACT 2
+	param set NAV_LOITER_RAD 50
+
+	param set RWTO_TKOFF 1
+	#param set FW_THR_SLEW_MAX 0.3 # fix takeoff failure for JSBsim in autonomous mission mode.
```

Comment:
Is this still a problem? If not, can you remove those?

Reply:
Removed!